jenkins-pipelines/feature-ms-ad-ldap-longevity-100gb-4h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
```

Comment:
Can we use eu-north-1 as default region for new jobs?  /Cc @roydahan

Reply:
I don't  mind.

Reply:
changed